src/components/GlobalPreferences/Notifications/SubscriptionFilters.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Button, DropDown, GU } from '@aragon/ui'
+import PropTypes from 'prop-types'
+
+const SubscriptionFilters = ({
+  organizations,
+  selectedOrganization,
+  onOrganizationChange,
+  apps,
+  selectedApp,
+  onAppChange,
+  events,
+  selectedEvent,
+  onEventChange,
+  onClearFilters,
+}) => {
+  return (
+    <div
+      css={`
+        width: 100%;
+        margin-bottom: ${1 * GU}px;
+        display: inline-grid;
+        grid-gap: ${1.5 * GU}px;
+        grid-template-columns: auto auto auto auto;
+      `}
+    >
+      <DropDown
+        placeholder="Organization"
+        header="Organization"
+        items={organizations}
+        selected={selectedOrganization}
+        onChange={onOrganizationChange}
+      />
+      <DropDown
+        placeholder="App"
+        header="App"
+        items={apps}
+        selected={selectedApp}
+        onChange={onAppChange}
+      />
+      <DropDown
+        placeholder="Event"
+        header="Event"
+        items={events}
+        selected={selectedEvent}
+        onChange={onEventChange}
+      />
+      {(selectedOrganization !== -1 ||
+        selectedApp !== -1 ||
+        selectedEvent !== -1) && (
+        <Button
+          css={`
+            justify-self: end;
+          `}
+          onClick={onClearFilters}
+        >
+          Clear Filters
```

Comment:
Clearing filters moves the filters sizes, maybe we should hide the button via opacity so that it takes up the space and this does not happen?
![clear filters](https://user-images.githubusercontent.com/3072458/63696425-fc5a6c00-c81a-11e9-8f30-7f604d2d49a4.gif)